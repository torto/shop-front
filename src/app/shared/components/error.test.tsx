import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Error from "./error";

test("render error", () => {
  const { asFragment } = render(<Error error="Test error" />);

  expect(asFragment()).toMatchSnapshot();
});
test("render without error", () => {
  const { asFragment } = render(<Error error="" />);

  expect(asFragment()).toMatchSnapshot();
});
