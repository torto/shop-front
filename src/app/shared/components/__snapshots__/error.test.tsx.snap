// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render error 1`] = `
<DocumentFragment>
  <div
    class="MuiSnackbar-root MuiSnackbar-anchorOriginTopRight"
  >
    <div
      class="MuiPaper-root MuiAlert-root MuiAlert-standardError MuiPaper-elevation0"
      direction="down"
      role="alert"
      style="opacity: 1; transform: scale(1, 1); transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiAlert-icon"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M11 15h2v2h-2zm0-8h2v6h-2zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
          />
        </svg>
      </div>
      <div
        class="MuiAlert-message"
      >
        <div
          class="MuiTypography-root MuiAlertTitle-root MuiTypography-body1 MuiTypography-gutterBottom"
        >
          <strong>
            Request Error
          </strong>
        </div>
        <strong>
          Message: 
        </strong>
        Test error
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`render without error 1`] = `<DocumentFragment />`;
